docs/api-client/README.md

```diff
@@ -1404,3 +1404,96 @@ Example
   ]
 }
 ```
+
+## Private Instances
+
+### getGithubToken
+
+Retrieves Github personal access token stored in your environment variables. Used for automated private instance creation to push yaml files (associated with private instance you're creating) to `openlaw-infra` repo.
+
+::: warning Authorization
+This resource can only be accessed by a logged in user with an `Admin` role as further explained in the [toAdminUser method](#toadminuser).
+:::
+
+```
+GET /getGithubToken
+```
+
+**Parameters**
+None.
+
+**Response**
+Returns Github personal access token stored in your environment variables.
+
+Example
+
+```
+32759f037558e0cjp67163kf1le25bc6kfbv2c61
+```
```

Comment:
Let's remove the example response for this one. The description of the response is sufficient.